packages/proposals

```diff
@@ -0,0 +1,53 @@
+1 PKG_PRIORITY
+
+None of the current scripts use the PKG_PRIORITY variable.
+All but three packages set PKG_PRIORITY="optional". busybox, dbus and systemd
+set PKG_PRIORITY="required".
+The effect of PKG_PRIORITY is unknown.
+
+Proposal:
+Deprecate the PKG_PRIORITY variable
+
+
+2 PKG_SITE, PKG_LONGDESC, PKG_SHORTDESC
+
+None of the current scripts use the PKG_SITE variable.
+create_addon is the only script that uses the PKG_LONGDESC and PKG_SHORTDESC
+variables.
+In other words, the PKG_LONGDESC and PKG_SHORTDESC are only used for
+addons. For other packages, the PKG_SITE, PKG_LONGDESC and PKG_SHORTDESC are
+merely informative and redundant.
+
+Proposal:
+Deprecate PKG_LONGDESC and/or PKG_SHORTDESC for normal packages
+
```

Comment:
PKG_LONGDESC is used for addon decriptions. can be renamed to whatever you think makes sense (PKG_ADDON_DESC is what I use) and dropped for non-addon packages. PKG_SHORTDESC is good to have but I already dropped it. I know the meaning of packages most people who care already do. PKG_SITE is there to track upstream. just my 2 cents